crates/ruff_linter/src/rules/ruff/snapshots/ruff_linter__rules__ruff__tests__range_suppressions.snap

```diff
@@ -261,12 +311,12 @@ help: Remove unused suppression
 
 
 RUF100 [*] Unused suppression (non-enabled: `E501`)
-  --> suppressions.py:48:5
+  --> suppressions.py:48:20
```

Comment:
I didn't notice this when reviewing your previous PR. 

I think we should create a single diagnostic for a single `disable/enable` pair. It''s okay to address this in a separate PR

Reply:
I think I need to learn/understand how to build diagnostics with multiple fixes.